docs/_documentations/mdt-che-setupregistries.md

```diff
@@ -1,25 +1,75 @@
+5. Click **Select a Push Registry** to determine which registry you use to push Codewind style projects to.
 
-You'll need the following information to complete the instructions to add the registries:
-  - Server: `<registry-to-push-images-to>`
-  - Username: `<Your username>`
-  - Password: `<Your password>`
+## Setting the Docker registry
+After creating a Codewind workspace, set the container registry to deploy your projects. When you create or add an existing project to Codewind, Codewind prompts you for the registry.
 
-For information about adding deployment registries to Che, consult our [registry documentation](image-registry-credentials.html).
+To change the registry in use, run the `Codewind: Set Deployment Registry` command in Theia to dynamically set a new registry for your workspace. <br>
 
-Next step: [Create Codewind workspace](mdt-che-createcodewindworkspace.html)
+![Set deployment registry](dist/images/che-docs/SetDockerRegistry-1.png){:height="90px" width="623px"}. <br>
+
+![Set deployment registry location](dist/images/che-docs/SetDockerRegistry-2.png){:height="85px" width="633px"}. <br>
+
+![Test deployment](dist/images/che-docs/SetDockerRegistry-3.png){:height="208px" width="801px"}. <br>
+
+To proceed, ensure that the registry credentials were added with Che. Codewind restarts with the changes added.
+
+## Codewind Docker registry guidance
+When you run Codewind on Kubernetes for Codewind style projects, Codewind uses [`buildah`](https://github.com/containers/buildah) to build container images from Dockerfiles and to push the images to an image registry that you specify. Use the **Image Registry Manager** in your Codewind IDE to configure the registry to use. 
+
+Use any registry as long as `buildah` and the node that Codewind is running on can resolve the registry hostname. If `buildah` cannot resolve the registry hostname, `buildah` cannot push your projects to the registry, and deployment on Kubernetes is prevented.
+
+## Examples:
+These examples show recommended values for common registries. The following deployment registries are tested and verified with Codewind:
+- Docker Hub:
+    - Address: `docker.io`
+    - Namespace: `<namespace>`
+    - Credentials: Docker Hub user name and password or access token
+    - **Note:** For Docker Hub, the `Namespace` value is likely to be your user name. 
+- Quay.io:
+    - Address: `https://quay.io`
+    - Namespace: `<namespace>`
+    - Credentials: Quay.io user name and encrypted password
+- Artifactory
+    - Address: `<artifactory-hostname>`
+    - Namespace: `<namespace>`
+    - Credentials: Artifactory user name and access token
+- OpenShift Registry
+    - Address: `docker-registry.default.svc:5000`
+    - Namespace: `<project>`
+    - Credentials: OpenShift user name and access token
+
+## Specifying image registries for Codewind 
```

Comment:
Maybe we should move this section up to be clear about when a registry is needed?